### PR TITLE
Add italian translation, fix #46

### LINKED ISSUE #46: Italian translation
I could contribute by adding the it_IT translation

### PR DESCRIPTION
**Pull request by mazzcris**
Resolves: https://github.com/blackus3r/post-worktime-logger/issues/46

### CODE REVIEW

**Comment by blackus3r** [comment]
Resolves: https://github.com/blackus3r/post-worktime-logger/issues/46

File: lang/post-worktime-logger-it_IT.po [hunk 1]
```diff
@@ -0,0 +1,183 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Post Worktime Logger v1.2.3\n"
```
Comment (blackus3r) [inline lang/post-worktime-logger-it_IT.po:3]: This seems to be the wrong version number? Did you checked out swimlane/1.4.0 or master?

**Comment by blackus3r** [comment]
Well done :)

File: lang/post-worktime-logger-it_IT.po (hunk 1 above)
Comment (mazzcris) [reply lang/post-worktime-logger-it_IT.po:3]: swimlane/1.4.0

**Pull request merged**